src/common/ownsql.h

```diff
@@ -152,6 +152,12 @@ class OCSYNC_EXPORT SqlQuery
         bindValueInternal(pos, value);
     }
 
+    void bindValue(int pos, const QByteArray &value)
```

Comment:
This is me or it's unused?

Reply:
@er-vin When calling a template `bindValue` method with `QByteArray` as an argument - we need to convert it to a QString via `QString::fromUtf8` - otherwise, the next logging line in this function will output something like `/x0/x2/x3` for non-Latin file/folder name. So yeah, this is used, and this function is chosen when `QByteArray` is passed as a second argument.

Reply:
Ah pffff... it's obviously an overload... stupid me. See, underpowered brain I said. ;-)

Reply:
Alright, reviewed all the calls to `bindValue` and that should be fine. Makes me wonder (once more, it's not the first time) why this code base is not using QString all the way for paths because then we wouldn't have that problem in the first place... this is especially true since we convert them all to utf8 anyway so they could be QString at that point...

Oh well... that would be much more invasive to do that switch everywhere so let's go for this.